src/components/PopoverDropdown/PopoverDropdown.styles.tsx

```diff
@@ -73,15 +74,21 @@ const top = css<TStyleProps>`
         position: absolute;
         content: '';
         bottom: 0;
+        left: 2rem;
         transform: translateX(${(p) => `${p.move}%`}) translateY(100%);
         border: ${arrowSize} solid transparent;
         border-top-color: ${color.blueDark};
     }
 `;
 
-const DivStyledArrow = styled.div<TStyleProps>`
-    z-index: 999;
-    ${(p) => (p.position ? getContainerStyleByPosition(p.position) : '')}
+export const DivStyledArrow = styled.div<TStyleProps>`
+    z-index: 1002;
+    width: 130%;
+    height: 10px;
+    top: 32px;
```

Comment:
nice work @Darrellbor 

there's this small issue on the popoverdropdown; it has an extra space between the arrow the dorpdown list to fix this issue; do this:

`top: 34px`

<img width="395" alt="Screenshot 2022-05-06 at 13 31 30" src="https://user-images.githubusercontent.com/16763860/167131822-55f38115-e528-43ec-8c01-174b69393929.png">

Reply:
nice catch @akinolu52 and very cool to provide a fix too. Good collab guys i live to see this 🤩